client/app/actions/account_action_creator.coffee

```diff
@@ -107,7 +135,13 @@ module.exports = AccountActionCreator = (dispatch, state) ->
                     type: ActionTypes.REMOVE_ACCOUNT_SUCCESS
                     value: {accountID}
 
-    discover: (domain, config) ->
+
+    # FIXME: Redux move all actionsCreator
+    # in an other scope
+    # that's why we can't call here
+    # other methods of this instance
+    # saga may be a solution for this case
+    discover: ({domain, config}) ->
```

Comment:
Actually, its my fault, you have to pass dispatch as an argument to AccountActionCreator
AccountActionCreator(dispatch).check {config, domain}

@m4dz : This needs to be discussed between
```
AccountActionCreator(dispatch).check {config, domain}
AccountActionCreator.check dispatch, {config, domain}
```